e2etest/manifests/mirror-server.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
I would at least split this into its own commit since it's not related to the feature being added.  You could also put it in its own PR so it can merge without blocking on the feature going in.